<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenuFrame</class>
 <widget class="QFrame" name="mainMenuFrame">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="m_aboutBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_inputBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Keys</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_settingsBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_newGameBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_quitBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_informationBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <action name="actionNewGame">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Game Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionInputSettings">
   <property name="text">
    <string>Keys</string>
   </property>
   <property name="toolTip">
    <string>Keys</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>m_newGameBtn</tabstop>
  <tabstop>m_settingsBtn</tabstop>
  <tabstop>m_inputBtn</tabstop>
  <tabstop>m_informationBtn</tabstop>
  <tabstop>m_aboutBtn</tabstop>
  <tabstop>m_quitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_newGameBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionNewGame</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_settingsBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionSettings</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionQuit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_informationBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionHelp</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_inputBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionInputSettings</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_aboutBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionAbout</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
